### PR TITLE
OC-9 added GUI

### PR DESCRIPTION
**Pull request by JornPrengerFormeel**

### CODE REVIEW

File: OurCalculator.pde [hunk 1]
```diff
@@ -1,11 +1,33 @@
+Button onebutton = new Button(25, 200, "1");
+Button twobutton = new Button(100, 200, "1");
+Button threebutton = new Button(175, 200, "1");
+Button fourbutton = new Button(25, 275, "1");
+Button fivebutton = new Button(100, 275, "1");
+Button sixbutton = new Button(175, 275, "1");
+Button sevenbutton = new Button(25, 350, "1");
+Button eightbutton = new Button(100, 350, "1");
+Button ninebutton = new Button(175, 350, "1");
```
Comment (joseph-allen) [inline OurCalculator.pde:9]: Operators as well +,x, etc

**Review by joseph-allen — CHANGES_REQUESTED** [review]

File: OurCalculator.pde (hunk 1 above)
Comment (JornPrengerFormeel) [reply OurCalculator.pde:9]: Will do in a few hours, had a short lunch break now back to class

Comment (joseph-allen) [reply OurCalculator.pde:9]: no problem, also remember to commit with the issue number, not just X for me to change :P

**Comment by joseph-allen** [comment]
@jorn600 you also have conflicts. Have a look into resolving a conflict but I appreciate it can be messy so if you can't do it I will. I would just like you to have a little try or be aware of the issue :)

File: OurCalculator.pde [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+
+
+
+
```
Comment (joseph-allen) [inline OurCalculator.pde]: don't add unnecessary spaces

**Review by joseph-allen — CHANGES_REQUESTED** [review]

File: OurCalculator.pde (hunk 2 above)
Comment (JornPrengerFormeel) [reply OurCalculator.pde]: Could you do that, im on my phone now. Ill do It when im homw

**Comment by joseph-allen** [comment]
@jorn600 this branch also has conflicts, I will fix for now as I understand the code but that won't always be the case :)

**Pull request merged**